CMakeLists.txt

```diff
@@ -23,8 +23,18 @@ include(ECMSetupVersion)
 include(ECMGenerateHeaders)
 include(ECMGeneratePkgConfigFile)
 include(ECMUninstallTarget)
+find_package(PkgConfig REQUIRED)
 
+pkg_check_modules(ZSTD REQUIRED IMPORTED_TARGET "libzstd")
 find_package(Fcitx5Utils REQUIRED)
+find_package(fmt REQUIRED)
+
+if (TARGET fmt::fmt-header-only)
```

Comment:
Why fmt is here? It seems no code using it.

Reply:
It's used in engine.cpp and never missing in dependency ...

https://github.com/fcitx/libime-jyutping/blob/9d9dba3f22d0b9f09a639556f6d4c9a9c51e5c97/src/engine/engine.cpp#L29

https://github.com/fcitx/libime-jyutping/blob/9d9dba3f22d0b9f09a639556f6d4c9a9c51e5c97/src/engine/engine.cpp#L95

Reply:
Please submit a separate PR for that.

Also target_link_libraries need to be fixed too

Reply:
https://github.com/fcitx/libime-jyutping/pull/8